scripts/load_safe_deployments.py

```diff
@@ -0,0 +1,75 @@
+import json
+
+import requests
+
+safe_deployments_base_url = (
+    "https://raw.githubusercontent.com/safe-global/safe-deployments/main/src/assets"
+)
+safe_deployments_path = "../gnosis/safe/safe-deployments/safe-deployments.json"
+
+deployments = [
+    {"version": "v1.0.0", "contracts_json": ["gnosis_safe.json", "proxy_factory.json"]},
+    {
+        "version": "v1.1.1",
+        "contracts_json": [
+            "create_and_add_modules.json",
+            "create_call.json",
+            "default_callback_handler.json",
+            "gnosis_safe.json",
+            "multi_send.json",
+            "proxy_factory.json",
+        ],
+    },
+    {"version": "v1.2.0", "contracts_json": ["gnosis_safe.json"]},
+    {
+        "version": "v1.3.0",
+        "contracts_json": [
+            "compatibility_fallback_handler.json",
+            "create_call.json",
+            "gnosis_safe.json",
+            "gnosis_safe_l2.json",
+            "multi_send.json",
+            "multi_send_call_only.json",
+            "proxy_factory.json",
+            "sign_message_lib.json",
+            "simulate_tx_accessor.json",
+        ],
+    },
+    {
+        "version": "v1.4.1",
+        "contracts_json": [
+            "compatibility_fallback_handler.json",
+            "create_call.json",
+            "safe.json",
+            "safe_l2.json",
+            "multi_send.json",
+            "multi_send_call_only.json",
+            "safe_proxy_factory.json",
+            "sign_message_lib.json",
+            "simulate_tx_accessor.json",
+        ],
+    },
+]
+
+
+def load_safe_deployments():
+    """
+    Get safe deployments addresses from https://github.com/safe-global/safe-deployments/ and write a json file
+    """
+    safe_deployments = {}
+    for deployment in deployments:
+        for contract_filename in deployment["contracts_json"]:
+            response = requests.get(
+                f"{safe_deployments_base_url}/{deployment['version']}/{contract_filename}"
+            )
+            safe_deployment = response.json()
+            safe_deployments[safe_deployment["version"]] = {
+                safe_deployment["contractName"]: safe_deployment["networkAddresses"]
+            }
+    # Write json file
```

Comment:
I would generate the file in `Python` instead of `json`, so we don't need to load it when using it

Reply:
Yes I was thinking the same, this PR is still in draft and working in progress, anyway I appreciate your suggestions :)